projects/packages/videopress/src/client/block-editor/hooks/use-sync-media/index.ts

```diff
@@ -145,11 +147,15 @@ function arrangeTracksAttributes(
  *
  * @param {object} attributes      - Block attributes.
  * @param {Function} setAttributes - Block attributes setter.
+ * @param {boolean} blockSettingsSaved  - Temporary attribute to track when block settings have been saved on native.
+ * @param {Function} setBlockSettingsSaved  - Temporary attribute to track manually control when settings are synchronised on native.
  * @returns {UseSyncMedia}      Hook API object.
  */
 export function useSyncMedia(
 	attributes: VideoBlockAttributes,
-	setAttributes: VideoBlockSetAttributesProps
+	setAttributes: VideoBlockSetAttributesProps,
+	blockSettingsSaved = false,
+	setBlockSettingsSaved = undefined
```

Comment:
Nitpick: The default value for `setBlockSettingsSaved` should be `undefined` when the prop is not passed, hence we could avoid setting the default value here.

Reply:
Thank you, changed in https://github.com/Automattic/jetpack/pull/29996/commits/dcd27ca81d52dc3ed76f27322fcdd83e88d49de0 🙇‍♀️